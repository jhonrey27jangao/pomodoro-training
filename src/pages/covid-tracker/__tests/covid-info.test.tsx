import React from "react";
import { render } from "@testing-library/react";
import CovidInfo from "../covid-info";

describe("Test for CovidInfo Component", () => {
  let info = {};
  beforeEach(() => {
    info = {
      cases: 0,
      deaths: 0,
      recovered: 0,
    };
  });
  it("should render CovidInfo component", () => {
    const { container } = render(<CovidInfo info={info} />);
    expect(container).toMatchSnapshot();
  });
});
