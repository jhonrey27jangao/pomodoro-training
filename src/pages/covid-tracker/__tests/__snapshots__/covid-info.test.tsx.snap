// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for CovidInfo Component should render CovidInfo component 1`] = `
<div>
  <div
    class="sc-AxjAm jzMSHk"
  >
    <div
      class="sc-AxirZ cgIrKz"
    >
      <div
        class="sc-AxiKw kCfraY"
      >
        <div
          class="top"
        >
          <div
            class="sc-AxhCb krsugi"
          >
            <span>
              <img
                alt="Infected"
                src="infected.png"
              />
            </span>
          </div>
          <p>
            Cases:
          </p>
          <h3>
            0
          </h3>
        </div>
        <div
          class="bottom"
        >
          <p>
            Tracked from: Github Live api 
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-AxirZ cgIrKz"
    >
      <div
        class="sc-AxiKw kCfraY"
      >
        <div
          class="top"
        >
          <div
            class="sc-AxhCb knOXta"
          >
            <span>
              <img
                alt=" deats"
                src="death.png"
              />
            </span>
          </div>
          <p>
            Deaths:
          </p>
          <h3>
            0
          </h3>
        </div>
        <div
          class="bottom"
        >
          <p>
            Tracked from: Github Live api 
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-AxirZ cgIrKz"
    >
      <div
        class="sc-AxiKw kCfraY"
      >
        <div
          class="top"
        >
          <div
            class="sc-AxhCb XTtAT"
          >
            <span>
              <img
                alt="Infected"
                src="recovered.png"
              />
            </span>
          </div>
          <p>
            Recoveries:
          </p>
          <h3>
            0
          </h3>
        </div>
        <div
          class="bottom"
        >
          <p>
            Tracked from: Github Live api 
          </p>
        </div>
      </div>
    </div>
    <div
      class="sc-AxirZ cgIrKz"
    >
      <div
        class="sc-AxiKw kCfraY"
      >
        <div
          class="top"
        >
          <div
            class="sc-AxhCb cpNgZK"
          >
            <span>
              <img
                alt="Infected"
                src="github.png"
              />
            </span>
          </div>
          <p>
            Api Info:
          </p>
          <a
            href="https://github.com/javieraviles/covidAPI"
          >
            Live Coronavirus API
          </a>
        </div>
        <div
          class="bottom"
        >
          <p>
            Tracked from: Github Live api 
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
